changelog.d/py312-setlocale-fail.infrastructure.md

```diff
@@ -0,0 +1 @@
+Fixed tests failing because of changes made to `locale.normalize` in Python 3.12.
```

Comment:
afaik these filenames are supposed to be prepended with `+` when not refering to issue numbers https://github.com/reef-technologies/b2-sdk-python/blob/master/CONTRIBUTING.md#changelog